showcase/korogenoest.md

```diff
@@ -0,0 +1,130 @@
+
+
+{% include image.html file="/recommended_dataflow.png" caption="An overview of SARS-CoV-2 raw reads and metadata upload steps (a-d) to ENA enabled by Galaxy. Adapted from [A SARS-CoV-2 sequence submission tool for the European Nucleotide Archive](https://doi.org/10.1093/bioinformatics/btab421) publication" alt="Galaxy ENA uploader takes raw reads as fastq files and metadata spreadsheet as an input. Then human reads are removed from the sequence data and the raw sequences gets uploaded to ENA along with the accompanying metadata. After variation analysis and consensus sequence workflow execution also consensus sequences get uploaded to ENA.
+" %}
+
+
+All the sequenced raw reads results were uploaded to the University of Tartu Galaxy instance. From there the data manager selected a batch and successfully sequenced samples into a working collection. Forward and reverse sequences were paired up and run through the workflow of dehumanisation of raw sequence reads (uses Metagen-FastQC). 
+
+In Galaxy ENA uploader metadata checklist (ERC000033) was marked and the filled-out template. The runs input format was selected from a paired collection (that was human sequence free), an affiliation center was added, and the ENA upload tool was run.
+
+As of March 2023, over 23 000 Estonian SARS-CoV-2 sequences have been uploaded to ENA. The majority were sequenced during the KoroGeno-EST studies (14 251), complemented by 9978 successfully sequenced samples from ECDC. 
+
+The KoroGeno-EST studies were conducted in cooperation with the University of Tartu, the Estonian Health Board and SYNLAB Eesti OÜ during 2021-2022. The study was approved by Research Ethics Committee of the University of Tartu. The [KoroGeno-EST-3 and KoroGeno-EST-2022 projects](https://kliinilinemeditsiin.ut.ee/et/sisu/eesti-sars-cov-2-taisgenoomide-jarjestamine-ja-analuus-korogeno-est-1est-2est-3) were financed by the Republic of Estonia's Ministry of Education and Research.
+
+## Considerations in Estonia
+
+- Estonia has a single national Health Board
+- e-Estonia and the X-road (e-Identity, e-Health Records, e-Governance) infrastructure allows easy data linking across data sources using the personal identification number.
+- There has been previous studies and cooperation between the Health Board and the University of Tartu in virus monitoring (HIV, Flu)
+- There were no additional local laws restraining the study
+- While the code of the dashboards, data handling is not publicly available the team is happy to consult researchers facing similar situation in the future. The best way is to send an email to [ELIXIR-Estonia](elixir@ut.ee).  The original [Galaxy workflows](https://workflowhub.eu/workflows?filter%5Bproject%5D=33&filter%5Bquery%5D=covid) adapted and used are publicly available. 
```

Comment:
```suggestion
- While the code of the dashboards, data handling is not publicly available the team is happy to consult researchers facing similar challenges. The best way is to send an email to [ELIXIR-Estonia](elixir@ut.ee).  The original [Galaxy workflows](https://workflowhub.eu/workflows?filter%5Bproject%5D=33&filter%5Bquery%5D=covid) adapted and used are publicly available. 
```
The link to the workflows points at a filtered search result of workflows. It means that as new workflows are added, the list will change. Is it worth it to create a Collection in WorkflowHub with the workflows used in this Showcase specifically and link to it?